package.json

```diff
@@ -10,52 +10,63 @@
     "dev": "vite",
     "build": "vite build",
     "preview": "vite preview",
-    "prettier": "prettier --write \"**/*.{js,jsx,css}\"",
-    "lint:prettier": "prettier -l \"**/*.{js,jsx,css}\"",
+    "prettier": "prettier --write \"**/*.{ts,tsx,css}\"",
+    "lint:prettier": "prettier -l \"**/*.{ts,tsx,css}\"",
     "lint": "yarn lint:prettier && yarn lint:js && yarn lint:css",
-    "lint:js": "eslint '**/*.{js,jsx}' --ignore-path .prettierignore",
-    "lint:css": "stylelint './frontend/**/*.css'"
+    "lint:js": "eslint '**/*.{ts,tsx}' --ignore-path .prettierignore",
+    "lint:css": "stylelint './frontend/**/*.css'",
+    "types": "tsc"
   },
   "dependencies": {
     "@babel/polyfill": "^7.0.0",
-    "@sentry/browser": "^7.17.4",
+    "@sentry/browser": "^7.16.0",
+    "@tanstack/react-query": "^4.14.3",
+    "@tanstack/react-table": "^8.5.22",
     "formik": "^2.2.9",
     "intl": "1.2.5",
     "js-cookie": "^3.0.1",
-    "luxon": "^3.1.0",
+    "luxon": "^3.0.4",
     "react": "^18.2.0",
-    "react-collapse": "^5.1.1",
     "react-csv": "^2.2.2",
     "react-dom": "^18.2.0",
     "react-image": "^4.0.3",
     "react-intl": "6.2.1",
     "react-loadable": "5.5.0",
     "react-motion": "^0.5.2",
-    "react-query": "^4.0.0",
-    "react-router-dom": "6.4.3",
-    "react-table": "^7.8.0",
+    "react-router-dom": "6.3.0",
     "react-test-renderer": "^18.2.0",
     "react-textarea-autosize": "^8.3.4",
-    "react-truncate": "^2.3.0",
     "recompose": "^0.30.0",
     "reselect": "4.1.7",
     "sanitize.css": "13.0.0",
     "styled-components": "5.3.6",
     "whatwg-fetch": "^3.6.2"
```

Comment:
Some of these should be bumped.

Reply:
@ollfkaih just made me aware, and I'll fix. Most likely because they were bumped while I was making this PR and the rebase thought the old versions were the correct ones